### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maryammdot**

### CODE REVIEW

File: staff/maryam-malek/cv-styled/style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+body {
+    background-color: grey;
+    box-sizing: border-box;
+}
+.main {
+    display: grid;
+    grid-template: 1fr 1fr 1fr / 10% 1fr 1fr 10%;
```
Comment (seand52) [inline staff/maryam-malek/cv-styled/style.css]: if you are going to more than one row or column with the same dimensions you can use:
repeat (3, 1fr) for example. saves you some typing!

File: staff/maryam-malek/cv/index.html [hunk 2]
```diff
@@ -4,76 +4,85 @@
...
+        </div>
+    </header>  
+
+    <main class="main">
+        <section class="section section--studies">
```
Comment (anasanjuan) [inline staff/maryam-malek/cv/index.html:28]: "studies" is element or modifier? if it is an element then with BEN we write  "section__studies", if it is a modifier is right :)

File: staff/maryam-malek/cv/index.html [hunk 3]
```diff
@@ -4,76 +4,85 @@
...
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../cv-styled/style.css">
     <title>Curriculum Vitae</title>
    
 </head>
 <body>
-        <header>
-            <h1>Richard House</h1>
-            <address>
+    <header class="header">
```
Comment (fedeCucchiararo) [inline staff/maryam-malek/cv/index.html:12]: Checkout the indentation. Also, using a `<header>` within a `<header>` might not be recommended. [Check out this link](https://stackoverflow.com/questions/16546482/header-inside-a-header-html5) :) Please, correct me if I am wrong.

**Comment by aleixcam** [comment]
good code! :)

File: staff/maryam-malek/cv/index.html (hunk 3 above)
Comment (aleixcam) [reply staff/maryam-malek/cv/index.html:12]: there isn't a header inside another

File: staff/maryam-malek/cv/index.html [hunk 4]
```diff
@@ -4,76 +4,85 @@
...
-            </li>
-        </ul>
-               
-     </section>
+                
+        </section>
+        <section class="section section--abilities">
+            <h2 class="section__title">Abilities</h2> 
+            <ul class="section__text">
+                <li>
```
Comment (HenryNovoa) [inline staff/maryam-malek/cv/index.html:68]: I would not add a ```<p>``` inside a ```<li>``` element here.

File: staff/maryam-malek/cv/index.html (hunk 2 above)
Comment (maryammdot) [reply staff/maryam-malek/cv/index.html:28]: I think in this case is modifier, cause I've made the elements to be text and title, and studies, experience and abilities as modifiers of section. I guess both possibilities could be ok.

File: staff/maryam-malek/cv/index.html (hunk 4 above)
Comment (aleixcam) [reply staff/maryam-malek/cv/index.html:68]: Maybe is not necessary, but neither is wrong

**Pull request merged**